arrpy/dtype.py

```
@@ -0,0 +1,85 @@
+    ----------
+    data : array-like
+        Input data
+    
+    Returns
+    -------
+    DType
+        Inferred data type
+    """
+    if not data:
+        return DEFAULT_FLOAT_TYPE
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)